moveit_ros/moveit_servo/src/servo_parameters.yaml

```diff
@@ -0,0 +1,350 @@
+    type: string,
+    default_value: "panda_arm",
+    description: "The name of the moveit move_group of your robot"
+  }
+
+  planning_frame: {
+    type: string,
+    default_value: "panda_link0",
+    description: "The name of moveit planning frame."
+  }
+
+  ee_frame_name: {
+    type: string,
+    default_value: "panda_link8",
+    description: "The name of end-effector frame of your robot"
+  }
+
+  robot_link_command_frame: {
+    type: string,
+    default_value: "panda_link0",
+    description: "The frame of incoming command, change this to your own value when using a different robot."
+  }
+
+  monitored_planning_scene_topic: {
+    type: string,
+    default_value: "/planning_scene",
+    description: "The name of the planning scene topic. \
+                  planning_scene_monitor::PlanningSceneMonitor::DEFAULT_PLANNING_SCENE_TOPIC"
+  }
+
+################################ OUTPUTS #########################################
+  status_topic: {
+    type: string,
+    default_value: "~/status",
+    description: "The topic to which the status will be published"
+  }
+
+  command_out_topic: {
+    type: string,
+    default_value: "/panda_arm_controller/joint_trajectory",
+    description: "The topic on which servo will publish the joint trajectory \
+                  Change this to the topic your controller requires."
+  }
+
+  command_out_type: {
+    type: string,
+    default_value: "trajectory_msgs/JointTrajectory",
+    description: "The type of command that servo will publish",
+    validation: {
+      one_of<>: [["trajectory_msgs/JointTrajectory", "std_msgs/Float64MultiArray"]]
+    }
+  }
+
+  num_outgoing_halt_msgs_to_publish: {
+    type: int,
+    default_value: 4,
+    description: "If 0, republish commands forever even if the robot is stationary. \
+                  Otherwise, specify num. to publish. Important because ROS may drop \
+                  some messages and we need the robot to halt reliably."
+  }
+
+################################ INPUTS  #############################
+  cartesian_command_in_topic: {
+    type: string,
+    default_value: "~/delta_twist_cmds",
+    description: "The topic on which servo will receive the twist commands"
+  }
+
+  joint_command_in_topic: {
+    type: string,
+    default_value: "~/delta_joint_cmds",
+    description: "The topic on which servo will receive the joint jog commands"
+  }
+
+  command_in_type: {
+    type: string,
+    default_value: "unitless",
+    description: "The unit of the incoming command.",
+    validation: {
+      one_of<>: [["unitless", "speed_units"]]
+    }
+  }
+
+  joint_topic: {
+    type: string,
+    default_value: "/joint_states",
+    description: "The topic on which joint states can be monitored"
+  }
+
+################################ GENERAL CONFIG #############################
+  enable_parameter_update: {
+    type: bool,
+    default_value: false,
+    description: "If true, continuous check for parameter update is enabled"
+  }
```

Comment:
What needs to be clearly communicated here is that right now, only 2 parameters are actually updated!